code/modules/security_levels/keycard authentication.dm

```diff
@@ -165,9 +165,10 @@
 			feedback_inc("alert_keycard_auth_stationRevoke",1)
 		if("Emergency Response Team")
 			if(is_ert_blocked())
-				to_chat(usr, "<span class='warning'>All Emergency Response Teams are dispatched and can not be called at this time.</span>")
+				atom_say("All Emergency Response Teams are dispatched and can not be called at this time.")
 				return
-			to_chat(usr, "<span class = 'notice'>ERT request transmitted.</span>")
+			atom_say("ERT request transmitted!")
+			command_announcer.autosay("ERT request transmitted.")
```

Comment:
This has no name argument. Resulting in `[Command]` says, "ERT Request Transmitted"`

Reply:
Oh, I suppose I could just put "Keycard Authentication Device" as the name.